en/developer_guide/logs.rst

```diff
@@ -0,0 +1,75 @@
+Logs
+====
+
+For logs we use Monolog. The Logs table is displayed in administration section in the backoffice. it displays the date and time, user's IP, user's name, the current site's name and the log's message.
+
+The LogBundle is in openorchestra-cmsbundle.
+
+Configuration
+-------------
+
+The logs's configuration is in app/config/log.yml. This file allows to declare the handlers. The handlers configure where save the logs the information type to record and channels.
+
+To save the logs in MongoDB, We need to define a handler with connection's information of the base and collection's name. Level indicates information's type to record in the logs. For example 200 it's for info but there is also debug, error, etc ...
+
+.. code-block:: yaml
+
+        mongo:
+            type: mongo
+            level: 200
+            mongo:
+                host: "localhost"
+                database: "%open_orchestra_cms.mongodb.database%"
+                collection: log
+            channels: [openorchestra]
+
+
+Channels defined the channels where are written the logs. If no channel is indicated then the logs are written in all channels. It's possible to
+put multiple channels or exclude one. The services who have the channel ``openorchestra`` write only in this chanel and our handler shows logs who are in the ``openorchestra`` channel.
+
+Processor
+---------
+
+The loggers who that allow record logs have the date and time but not IP address and user name. For save IP address and user name, we have created a processor.
+In the processor tags, we spent the channel ``openorchestra``. So, for each log written in this channel he spends into the processor.
+
+The processor must be tagged by:
+
+.. code-block:: yaml
+
+    tags:
+        - { name: monolog.processor, method: processRecord, channel: openorchestra }
+
+Logger
+------
+Services that write the logs must have in parameter ``Symfony\Bridge\Monolog\Logger``
```

Comment:
services that write the logs take a ``Symfony\Bridge\Monolog\Logger`` in parameter and must be tagged by `monolog.logger``